### ISSUE TITLE
AttributeError: "'NoneType' object has no attribute 'find_all'" 

### ISSUE THREAD

**Issue by kenneyhe** [sealemar/BingRewards#310]
symptom:
-> rewards = bdp.parseDashboardPage(bingRewards.getDashboardPage(), bingCommon.ACCOUNT_URL)
AttributeError: "'NoneType' object has no attribute 'find_all'"

page is always wrong with:
JavaScript required to sign in

> /Users/Shared/BingRewards/pkg/bingRewards.py(116)getDashboardPage()
-> if page.find('JavaScript required to sign in') == -1:
(Pdb) p page
'<html><head><noscript>JavaScript required to sign in<meta http-equiv="Refresh" content="0; URL=https://login.live.com/jsDisabled.srf?mkt=EN-US&lc=1033"/></noscript><title>Continue</title><script type="text/javascript">function OnBack(){}function OnNext(){}function DoSubmit(){var subt=false;if(!subt){subt=true;document.fmHF.submit();}}</script></head><body onload="javascript:DoSubmit();"><form name="fmHF" id="fmHF" action="https://account.microsoft.com/auth/complete-silent-signin?ru=https%3a%2f%2faccount.microsoft.com%2frewards%2fdashboard%3frefd%3dwww.bing.com&wa=wsignin1.0" method="post" target="_self"><input type="hidden" name="pprid" id="pprid" value="4943fa87ae9bb3e6"><input type="hidden" name="NAP" id="NAP" value="V%3D1.9%26E%3D144c%26C%3D8e-liPuHAx1NRWV5ktRcOB4Bxo12g-SfuSO8TVU3CWUhxm5--b4VQw%26W%3D1"><input type="hidden" name="ANON" id="ANON" value="A%3D5FDC0D231AC090A4E6DDC126FFFFFFFF%26E%3D14a6%26W%3D1"><input type="hidden" name="t" id="t" value="GABAAgMAAAAEgAAADAAgb56XKI1nongWCkGGJKR+0JkGd9vopZV0rDEaHAvIJRwAARtk2FjIuJAaLHP0wjwwM9stvSK/IjNRpyvdP/NDm/cXFNKffLd48Ma89OY9iTUPFCij7QDDJdWZaza5Pg+nDpUvKVyAY5IkDzaxlrIq0Hok6ufv4nEyY2+rSLS7+Pb7uoZBMGlS7X8tlbz4NnaF3HzNs7YTtDjUYf1M2vmDIuv7W8A1IoJVvHsOXHR2cXWHP2ip4BazeyeMt6spwg7QVqmVM6Y4m0pkSd6hM37bWniOsVuN5EiUANt8nE5dxxvlnMTv8AWbw4JW9qjJGCFirDoQwxz8mZnsrwdwkRO8e8KL+D5ayO0qIbMg7kv6+bfJj2wEKniCS6izDtC36yhWTNQNAXQADQH+fwMA0VNr2rvuP1qM7j9aOncEAAoQIAAQAABZAAAeMjUwMTM5MDcwNyVxcS5jb21AcGFzc3BvcnQuY29tAAAAHlVTAAU5NDEzMgAAAAAECQIAAHt2ZkAABkEABVRSQUNZAANUQU0EXwAAAAAAAAAAAAAAAAAAAACum7PmSUP6hwAAu+4/Wo2VtloAAAAAAAAAAAAAAAAOADczLjkzLjE0MC4yMzcABQEAAAAAAAAAAAAAAAABBAAAAAAAAAAAAAAAAAAAAMueilcwQ0/8AAAAAAAAAAAAAAAAAAAAAAAAEgAyNTAxMzkwNzA3QHFxLmNvbQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"></form></body></html>'

**Comment by specu** [comment]
is this with main tree or @baltimoron fork? I had that before, but after clean pull of latest version it all went away.

**Comment by baltimoron** [comment]
Are you running on linux or windows? For some reason the indentation in that file looks correct for me on linux but wrong on windows.

**Comment by kenneyhe** [comment]
@baltimoron I am running on MacOSX 10.11.4.  @specu It is with main tree.  It only happens to one accounts and the rests of my other 6 accounts are ok with the same code

**Comment by specu** [comment]
was that account converted to new dashboard?

**Comment by kenneyhe** [comment]
I see this 2 years old account having new dashboard. I did not choose new dashboard.

**Comment by specu** [comment]
you have to use balitmoron's fork to run that account

**Comment by kenneyhe** [comment]
@specu Yes. balitmoron's master fork fixed it. which commit fix it?  I want to merge it to my fork. :)

**Comment by specu** [comment]
I'm running converted account from a different directory where I have baltimiron's up-to-date fork. Works fine for me.

**Comment by baltimoron** [comment]
@kennyhe all the commits on my fork are related to this issue so just grab the latest. FYI it is currently identifying the hits and hitting them but not getting any points for it.

**Comment by specu** [comment]
@kenneyhe I believe this is now fixed. Please close this issue if this is fixed for you.
